eventmesh-trace-plugin/build.gradle

```diff
@@ -13,4 +13,5 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
\ No newline at end of file
+ */
+spotlessJava.enabled false
```

Comment:
This is because the execution of spotless tasks on some parent projects needs to display the declaration of dependencies, and this declaration is often more complex, and it needs to change frequently with the increase of child projects, but there is no code that needs to be converted on the parent project.